homeassistant/components/media_player/sonos.py

```diff
@@ -891,10 +891,7 @@ def set_volume_level(self, volume):
     def set_shuffle(self, shuffle):
         """Enable/Disable shuffle mode."""
         self._shuffle = shuffle
```

Comment:
Thanks for your comments armills. The recent commit have adressed your suggestions